src/App.js

```diff
@@ -33,12 +34,21 @@ import {
 
 import monitorNetwork from './debugging/network';
 import handleDeeplink from './handlers/deeplinks';
+import {
+  getUserBackupState,
+  saveUserBackupState,
+} from './handlers/localstorage/globalSettings';
 import DevContextWrapper from './helpers/DevContext';
+import BackupStateTypes from './helpers/backupStateTypes';
 import { withAccountSettings } from './hoc';
 import { registerTokenRefreshListener, saveFCMToken } from './model/firebase';
 import * as keychain from './model/keychain';
+import { loadAddress } from './model/wallet';
 import { Navigation } from './navigation';
+// eslint-disable-next-line import/no-cycle
```

Comment:
Can you move the context to another file instead of making a cycle?

Reply:
Good call!